contracts/RelayHub.sol

```diff
@@ -114,24 +117,30 @@ contract RelayHub is IRelayHub {
         emit Withdrawn(account, dest, amount);
     }
 
-    function verifyGasLimits(
-        uint256 paymasterMaxAcceptanceBudget,
+    function calldataGasCost(uint256 length) public override view returns (uint256) {
```

Comment:
Does it have to be part of an interface? I think this method can be internal.

Reply:
Currently it's also being used in the server, and in general I believe that it might vary between hubs, and might be needed outside, whether on or off chain.

Reply:
accepted